### PR TITLE
added german translations, fixes issue #80

### PR DESCRIPTION
**Pull request by root-ansh**
- added german translations in `values-de` folder
-  fixes issue #80
- My Android studio is currently blasted up, so made the folder manually. please check if its working correctly
- I am not a native German Speaker. All the Translations are done using Google Translate and with the best of my abilities. Please consider

#Happy HacktoberFest 2019!

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+  ~     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~     GNU General Public License for more details.
+  ~
+  ~     You should have received a copy of the GNU General Public License
+  ~     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+  -->
+
+<resources>
+    <string name="app_name">Ara</string>
+    <string name="action_settings">die Einstellungen</string>
```
Comment (Idrinth) [inline app/src/main/res/values-de/strings.xml:19]: for a menu you wouldn't usually add the article, likely this should just be "Einstellungen"

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+    <string name="action_about">Über</string>
+    <string name="greet">Hallo, kann ich helfen?</string>
```
Comment (Idrinth) [inline app/src/main/res/values-de/strings.xml:21]: Usually you'd ask how you can help, so "Hallo, wie kann ich helfen?"

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+    <string name="greet1">Hallo, @string/greet</string>
+    <string name="blank" />
+    <string name="title_activity_settings">die Einstellungen</string>
```
Comment (Idrinth) [inline app/src/main/res/values-de/strings.xml:24]: As above, usually not used with the article included

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+  <string name="new_app_widget2_title">WIDGET KOMMT BALD</string>
+
+    <string name="pref_title_display_name">Anzeigename</string>
+    <string name="pref_default_display_name">John Smith</string>
+
+    <string name="pref_title_add_friends_to_messages">Freunde zu Nachrichten hinzufügen</string>
+    <string-array name="pref_example_list_titles">
+        <item>Immer</item>
+        <item>Wenn möglich</item>
+        <item>noch nie</item>
```
Comment (Idrinth) [inline app/src/main/res/values-de/strings.xml:46]: only "nie", "noch nie" means not yet

File: app/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+    <string name="pref_title_social_recommendations">Aktivieren Sie soziale Empfehlungen</string>
+    <string name="pref_description_social_recommendations">
+	
+Empfehlungen für Kontaktpersonen basierend auf Ihrem Nachrichtenverlauf
+    </string>
+  <string name="botton_menu_title">Zuhause</string>
+  <string name="new_app_widget2_title">WIDGET KOMMT BALD</string>
+
+    <string name="pref_title_display_name">Anzeigename</string>
+    <string name="pref_default_display_name">John Smith</string>
```
Comment (Idrinth) [inline app/src/main/res/values-de/strings.xml:40]: The german equivalent is "Max Mustermann"

File: app/src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+        "Positionieren Sie den FAB beispielsweise an einer Seite des Kartenstroms, damit der FAB nicht stört."
+        "Wenn ein Benutzer versucht, eine der Karten aufzuheben. \n\n"
+    </string>
+    <string name="image">Kamera</string>
+    <string-array name="front">Hallo @array/pref_example_list_values</string-array>
+
+    <string name="title_activity_setup">Konfiguration</string>
+    <string name="prompt_email">Email</string>
+    <string name="prompt_password">Passwort</string>
+    <string name="action_sign_in">Einloggen oder Registrieren</string>
```
Comment (Idrinth) [inline app/src/main/res/values-de/strings.xml:201]: shouldn't it be either "Einloggen" or "Anmelden" in both cases?

**Review by Idrinth — CHANGES_REQUESTED** [review]
just a few translation issues

File: app/src/main/res/values-de/strings.xml (hunk 3 above)
Comment (root-ansh) [reply app/src/main/res/values-de/strings.xml:24]: So, for line `#22` ,it should be just `@string/greet` ? or should i replace `<string name="title_activity_settings">die Einstellungen</string>` with `<string name="title_activity_settings">Einstellungen</string>`?

File: app/src/main/res/values-de/strings.xml (hunk 5 above)
Comment (root-ansh) [reply app/src/main/res/values-de/strings.xml:40]: for the names or for `line #37`?

File: app/src/main/res/values-de/strings.xml (hunk 6 above)
Comment (root-ansh) [reply app/src/main/res/values-de/strings.xml:201]: Sorry, Can't comment. I am not much knowledgeable and relied on google translate for this. Your call

File: app/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (root-ansh) [reply app/src/main/res/values-de/strings.xml:19]: changed. you would be seeing the change in new commit

File: app/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (root-ansh) [reply app/src/main/res/values-de/strings.xml:21]: Sure. Thanks for the review

File: app/src/main/res/values-de/strings.xml (hunk 4 above)
Comment (root-ansh) [reply app/src/main/res/values-de/strings.xml:46]: Sure. Changed that

File: app/src/main/res/values-de/strings.xml (hunk 5 above)
Comment (Idrinth) [reply app/src/main/res/values-de/strings.xml:40]: for the default name - not sure if changing is necessary

File: app/src/main/res/values-de/strings.xml (hunk 6 above)
Comment (Idrinth) [reply app/src/main/res/values-de/strings.xml:201]: I hoped to not be the only one checking :) both are correct, personally I prefer "Anmelden", but that is just my preference

File: app/src/main/res/values-de/strings.xml (hunk 3 above)
Comment (Idrinth) [reply app/src/main/res/values-de/strings.xml:24]: the later, so "Einstellungen" is fine. thanks for your work btw :)

**Pull request merged**